.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,20 @@
+name: helm-release
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  call-update-helm-repo:
+    uses: grafana/helm-charts/.github/workflows/update-helm-repo.yaml@main
+    with:
+      charts_dir: production/helm
+      cr_configfile: production/helm/cr.yaml
+      ct_configfile: production/helm/ct.yaml
+    secrets:
+      helm_repo_token: ${{ secrets.GH_BOT_ACCESS_TOKEN }}
+      # See https://github.com/grafana/helm-charts/blob/main/INTERNAL.md about this key
+      gpg_key_base64: ${{ secrets.HELM_SIGN_KEY_BASE64 }}
```

Comment:
@trevorwhitney do you know which key we used to use? There's a public key checked into `production/helm`. Should we generate a new key?

Reply:
yikes, no, I have no idea. is there a way we can reverse engineer what key was being used in the `grafana/helm-charts` repo? maybe we could ask @krajorama and @Logiraptor what they did when the moved the mimir helm chart?

Reply:
I'll have to defer to @krajorama on what was done. I can see a signing key exists in the settings of the mimir repo, but I can't view the secret contents.

Reply:
We are still missing the key.